planner/core/logical_plan_builder.go

```diff
@@ -717,11 +715,6 @@ func (ds *DataSource) setPreferredStoreType(hintInfo *tableHintInfo) {
 }
 
 func isTiFlashComputeNodeAvailable(ctx sessionctx.Context) bool {
```

Comment:
It is always true. Can we use a const variable?

Reply:
Delete `isTiFlashComputeNodeAvailable`. Will check tilfash_compute when dispatch task instead of explain.